docs/components/connectors/custom-built-connectors/connector-sdk.md

```diff
@@ -287,14 +307,26 @@ The Connector runtime environment initializes the context and allows the followi
 - Fetch and deserialize the input data as shown in **(1)**. Refer to the [input data](#outbound-connector-input-data) section for details.
 - Execute the Connector's business logic as shown in **(2)**.
 
+###### Basic errors
+
 If the Connector handles exceptional cases, it can use any exception to express technical errors. If a technical
 error should be associated with a specific error code, the Connector can throw a `ConnectorException` and define
 a `code` as shown in **(3)**.
 We recommend documenting the list of error codes as part of the Connector's API. Users can build on those codes
-by creating [BPMN errors](/components/connectors/use-connectors/index.md#bpmn-errors) in their Connector configurations.
+by creating [BPMN errors](/components/connectors/use-connectors/index.md#bpmn-errors) in their Connector configurations.<br/>
+
+###### Retryable errors
```

Comment:
```suggestion
##### Retryable errors
```

See comment above. I'm not clear on what should be a subsection.

Reply:
Also, is this based on https://docs.spring.io/spring-retry/docs/api/current/org/springframework/retry/annotation/Retryable.html?